public/app/features/alerting/unified/components/amroutes/MuteTimingForm.tsx

```diff
@@ -109,11 +111,12 @@ const MuteTimingForm = ({ muteTiming, showError }: Props) => {
         disabled
         dataSources={alertManagers}
       />
+      {provenance && <ProvisioningAlert type="mute timing" />}
       {result && !loading && (
         <FormProvider {...formApi}>
           <form onSubmit={formApi.handleSubmit(onSubmit)} data-testid="mute-timing-form">
             {showError && <Alert title="No matching mute timing found" />}
-            <FieldSet label={'Create mute timing'}>
+            <FieldSet label={'Create mute timing'} disabled={Boolean(provenance)}>
```

Comment:
I'm not a fan of `Boolean(string)` as it returns true for `Boolean("false")` :/

Reply:
Good point, though if the resource has not been provisioned the API _should_ return an undefined value. Though I imagine this would be confusing if users manually set the provenance (via the API) to "false" but that feels like a case of PEBKAC :D

Reply:
Ok, that makes much more sense now :)